doc/dev/overview.rst

```diff
@@ -0,0 +1,104 @@
+**************
+Code Structure
+**************
+
+
+Overview
+########
+
+The **autodE** code base is structured around a :code:`Reaction` class, due to
+the initial singular goal of calculating reaction profiles.
+
+.. image:: ../common/reaction_simple_uml.svg
+   :target: ../_images/reaction_simple_uml.svg
+   :width: 550
+   :align: center
+
+|
+|
+
+The key attributes include :code:`reacs` and :code:`prods` as individual molecules
+comprising the reactants and products of the reaction. Transition states connecting
+reactants and products are held in :code:`tss`. From these attributes are generated
```

Comment:
"From these attributes are generated properties," - Doesn't seem to make sense to me? Do you mean "From these, property attributes are generated, including..."